src/scripts/main.js

```diff
@@ -5,4 +5,22 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach(person => {
+  const tr = document.createElement('tr');
+
+  person.age = person.died - person.born;
+
+  person.century = Math.ceil(person.died / 100);
+
+  tr.insertAdjacentHTML('afterbegin', `
+  <td>${person.name}</td>
+  <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
```

Comment:
nowadays if sex is not male it doesn't mean it's female) so it's always better to do explicit check 
```
let sex = '';

switch (person.sex) {
  case 'm':
    sex = 'Male';
  case 'f':
    sex = 'Female';
}
```